srcpkgs/edk2-shell/template

```diff
@@ -0,0 +1,75 @@
+# Template file for 'edk2-shell'
+_brotli_ver=1.0.7
+pkgname=edk2-shell
+version=202005
+revision=1
+wrksrc=edk2-edk2-stable202005
+hostmakedepends="nasm"
+makedepends="libuuid-devel python3"
+short_desc="EDK2 UEFI Shell"
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="BSD-2-Clause-Patent"
+homepage="https://github.com/tianocore/edk2"
+distfiles=" https://github.com/tianocore/edk2/archive/edk2-stable${version}.tar.gz
+			https://github.com/google/brotli/archive/v${_brotli_ver}.tar.gz"
```

Comment:
We ships `brotli`. Does it work with system brotli?

Reply:
If you see the PKGBUILD, two packages are created (edk2-shell and edk2-ovmf). How do I merge into voidlinux? edk2-shell is not a subpackage of edk2-ovmf.
Another question: I have not seen that a pull request [17225](https://github.com/void-linux/void-packages/pull/17225) already exists that deals with this functionality.
What do we do? it looks old ......